docs/minio-client-complete-guide.md

```diff
@@ -242,35 +202,30 @@ mc --json ls play
 ```
 
 ### Option [--no-color]
-
 This option disables the color theme. It useful for dumb terminals.
 
 ### Option [--quiet]
-
 Quiet option suppress chatty console output.
 
 ### Option [--config-folder]
-
 Use this option to set a custom config path.
 
 ### Option [ --insecure]
-
 Skip SSL certificate verification.
 
 ## 7. Commands
 
 |   |   | |
 |:---|:---|:---|
-|[**ls** - List buckets and objects](#ls)   |[**mb** - Make a bucket](#mb)   | [**cat** - Concatenate an object](#cat)  |
-|[**cp** - Copy objects](#cp)   | [**rm** - Remove objects](#rm)  | [**pipe** - Pipe to an object](#pipe)  |
-| [**share** - Share access](#share)  |[**mirror** - Mirror buckets](#mirror)   |[**diff** - Diff buckets](#diff)   |
-|[**policy** - Set public policy on bucket or prefix](#policy)   |[**session** - Manage saved sessions](#session)   | [**config** - Manage config file](#config)  |
-| [**watch** - Watch for events](#watch)   | [**events** - Manage events on your buckets](#events)   |   | 
+|[**ls** - List buckets and objects](#ls)   |[**mb** - Make a bucket](#mb)  | [**cat** - Concatenate an object](#cat)  |
```

Comment:
Not really necessary, but it would be visually easy to see if the texts and table cell separators, `|`, are aligned as much as possible.

Reply:
it s harder to do markdown you should read after rendered not like this.